docs/src/main/asciidoc/includes/attributes.adoc

```diff
@@ -24,16 +24,18 @@ ifndef::attributes-included[]
 :imagesdir: {rootdir}/images
 
 :helidon-version: 4.1.0-SNAPSHOT
-:helidon-version-is-release: true
+:helidon-version-is-release: false
 
 :sourcedir: {rootdir}/../java/io/helidon/docs
 
 ifeval::["{helidon-version-is-release}" != "true"]
 :helidon-github-tree-url: https://github.com/oracle/helidon/tree/main
+:helidon-github-examples-url: https://github.com/helidon-io/helidon-examples/tree/helidon-4.x/examples
```

Comment:
Not super-important, but at some point the `helidon-4.x` part of this could be an attribute like `helidon-version` that is used a few lines below and declared right next to it so someone who revises this for 5.x would be more likely to update both correctly.